packages/trader/src/sass/app/modules/trading-mobile.scss

```
@@ -185,7 +185,19 @@
                 }
             }
         }
-        &__amount {
+
+        &__amount,
+        &__stake {
+            &-container {
+                margin: 0.4rem 0 -0.7rem;
```

```suggestion
                margin: 0.4rem 0 -0.7rem;
```
👀  negative rem? why? can it be solved by using a positive value on opposite side?

---

I'm sorry, but below there is a `trade-params__amount-keypad` with margin-top 0.8 rem and I can't change this value as it will affect all trade types on mobile with this keypad (because in all the other trade types keypad will be closure to the top, then it should be)

---

Ah i see. LGTM